File: client/src/app/_models/hr/intervwItem.ts
import { IIntervwItemCandidate } from "./intervwItemCandidate";

export interface IIntervwItem {  
     visible: boolean;
     id: number;
     intervwId: number;
     interviewVenue: string;
     venueAddress: string;
     venueAddress2: string;
     venueCityAndPIN: string;
     siteRepName: string;
     sitePhoneNo: string;
     orderNo: number;
     orderItemId: number;
     professionId: number;
     professionName: string;
     categoryRef: string;
     interviewMode: string;
     interviewerName: string;
     estimatedMinsToInterviewEachCandidate: number;
     interviewItemCandidates: IIntervwItemCandidate[];
}

File: api/Entities/Admin/IntervwItem.cs
using System.ComponentModel.DataAnnotations;

namespace api.Entities.Admin
{
    public class IntervwItem: BaseEntity
    {
        public int IntervwId { get; set; }
        public string InterviewVenue { get; set; }
        public DateTime InterviewScheduledFrom {get; set;}
        public string VenueAddress { get; set; }
        public string VenueAddress2 { get; set; }
        public string VenueCityAndPIN { get; set; }
        public string SiteRepName { get; set; }
        public string SitePhoneNo { get; set; }
        public int OrderItemId { get; set; }
        public int ProfessionId { get; set; }
        public string ProfessionName { get; set;}   
        [MaxLength(10)]
        public string CategoryRef { get; set; }
        public string InterviewMode { get; set; }
        public string InterviewerName { get; set; }
        public int EstimatedMinsToInterviewEachCandidate{ get; set; }=25;
        public ICollection<IntervwItemCandidate> InterviewItemCandidates { get; set; }
    }
}